parts/k8s/kubernetescustomscript.sh

```diff
@@ -86,9 +86,7 @@ installKubeletAndKubectl
 ensureRPC
 createKubeManifestDir
 if [[ "${SGX_NODE}" = true ]]; then
-    if $FULL_INSTALL_REQUIRED; then
-        installSGXDrivers
-    fi
+    installSGXDrivers
```

Comment:
This change means `installSGXDrivers` is called always during provisioning, even if the drivers were already installed on the VHD image. That seems like a de-optimization. Does this fix something?

Reply:
The `FULL_INSTALL_REQUIRED` does not apply to SGX driver installation. It was copy-pasted by mistake. Cleaning it up.